HIP/hip-260.md

```diff
@@ -53,9 +55,11 @@ transactions.
 
 ### Add ContractStorageChanges to ContractFunctionResult
 
-Add and populate a new field `stateChanges` of type `ContractStorageChagnes` to
-the `ContractFunctionResult` message. This will be populated whenever a
-transaction causes contract storage states to change.
+Add and populate a new repeated field `stateChanges` of
+type `ContractStorageChanges` to the `ContractFunctionResult` message. This will
```

Comment:
Can we show this protobuf snippet instead of just describing it?

Reply:
done

Reply:
fixed, set to `ContractStateChange` in header and text